source/geod24/concurrency.d

```diff
@@ -1061,9 +1064,13 @@ protected:
 
     protected class FiberCondition : Condition
     {
-        this() nothrow
+        /// The mutex with which this condition will be associated.
+        private Mutex _mutex;
```

Comment:
https://github.com/dlang/druntime/blob/056cbdfe9c3aade1031a040aa66c18e79200f254/src/core/sync/condition.d#L155-L158

Reply:
I need to modify it to `this.mutex_nothrow`.